diff --git a/node_modules/react-native-screens/android/CMakeLists.txt b/node_modules/react-native-screens/android/CMakeLists.txt
index 86ec079..f6b55ba 100644
--- a/node_modules/react-native-screens/android/CMakeLists.txt
+++ b/node_modules/react-native-screens/android/CMakeLists.txt
@@ -78,3 +78,8 @@ else()
         android
     )
 endif()
+
+find_library(LIBCXX_SHARED c++_shared)
+if(LIBCXX_SHARED)
+    target_link_libraries(rnscreens ${LIBCXX_SHARED})
+endif()
diff --git a/node_modules/react-native-screens/android/src/main/jni/CMakeLists.txt b/node_modules/react-native-screens/android/src/main/jni/CMakeLists.txt
index 1b26dac..eaa1329 100644
--- a/node_modules/react-native-screens/android/src/main/jni/CMakeLists.txt
+++ b/node_modules/react-native-screens/android/src/main/jni/CMakeLists.txt
@@ -45,6 +45,7 @@ if(ReactAndroid_VERSION_MINOR GREATER_EQUAL 76)
     ReactAndroid::reactnative
     ReactAndroid::jsi
     fbjni::fbjni
+    c++_shared
   )
 else()
   target_link_libraries(
@@ -65,6 +66,7 @@ else()
     rrc_view
     turbomodulejsijni
     yoga
+    c++_shared
   )
 endif()
 
@@ -87,4 +89,3 @@ target_include_directories(
  PUBLIC
  ${CMAKE_CURRENT_SOURCE_DIR}
 )
-
